src/litdata/processing/utilities.py

```diff
@@ -249,3 +249,37 @@ def extract_rank_and_index_from_filename(chunk_filename: str) -> Tuple[int, int]
     index = int(chunk_filename[1].split(".")[0])
 
     return rank, index
+
+
+def remove_uuid_from_filename(filepath: str) -> str:
+    """Remove the unique id from the filepath. Expects the filepath to be in the format
+    `checkpoint-<rank>-<uuid>.json`.
+
+    e.g.: `checkpoint-0-9fe2c4e93f654fdbb24c02b15259716c.json`
+        -> `checkpoint-0.json`
+
+    """
+
+    if not filepath.__contains__(".checkpoints"):
+        return filepath
+
+    # uuid is of 32 characters, '.json' is 5 characters and '-' is 1 character
+    return filepath[:-38] + ".json"
+
+
+def downloadDirectoryFromS3(bucketName: str, remoteDirectoryName: str, localDirectoryName: str) -> str:
+    s3_resource = boto3.resource("s3")
+    bucket = s3_resource.Bucket(bucketName)
```

Comment:
It is good to keep python style to use `bucket_name` vs `bucketName`. This applies in several places.

Reply:
sure.